### PR TITLE
chore: Wait a little after cluster comes up

### PR DESCRIPTION
**Pull request by bradbeam**
Signed-off-by: Brad Beam <brad.beam@talos-systems.com>

### CODE REVIEW

File: hack/test/osctl-cluster-create.sh [hunk 1]
```diff
@@ -53,6 +53,9 @@ run "timeout=\$((\$(date +%s) + ${TIMEOUT}))
 	   sleep 5
 	 done"
 
+## Give a moment for things to stabilize before applying manifests
+sleep 10
```
Comment (andrewrynhard) [inline hack/test/osctl-cluster-create.sh]: Not for this PR, but curious what we can wait on via `kubectl wait` before we apply this. Only started seeing this after 1.16 move.

File: .drone.yml [hunk 2]
```diff
@@ -488,6 +488,7 @@ steps:
       BUILDKIT_HOST: ${BUILDKIT_HOST=tcp://buildkitd.ci.svc:1234}
       BINDIR: /usr/local/bin
       PLATFORM: azure
+      CONFORMANCE: run
```
Comment (bradbeam) [inline .drone.yml]: Accidentally dropped these in #952

File: hack/test/conformance.sh [hunk 3]
```diff
@@ -7,19 +7,23 @@ source ./hack/test/e2e-runner.sh
 echo "Master CIS Conformance:"
 e2e_run "export KUBECONFIG=${KUBECONFIG}-${PLATFORM}-capi
          kubectl apply -f /e2emanifests/cis-kube-bench-master.yaml
-         kubectl wait --timeout=60s --for=condition=complete job/kube-bench-master > /dev/null
+         kubectl wait --timeout=300s --for=condition=complete job/kube-bench-master > /dev/null
```
Comment (bradbeam) [inline hack/test/conformance.sh]: Seeing issues in azure e2e where 60s isn't enough.

File: hack/test/e2e-integration.sh [hunk 4]
```diff
@@ -85,8 +86,7 @@ e2e_run "timeout=\$((\$(date +%s) + ${TIMEOUT}))
 		 done"
 
 ## Run conformance tests if var is not null
-if [ -n "${CONFORMANCE}" ]
-then
+if [ ${CONFORMANCE:-"dontrun"} == "run" ]; then
```
Comment (bradbeam) [inline hack/test/e2e-integration.sh]: -n conformance fails with our options above ( set -eou pipefail ), so we need to ensure it has some default value so they can be properly skipped

**Pull request merged**